exo-go/src/logger/index.go

```diff
@@ -0,0 +1,124 @@
+package logger
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/Originate/exosphere/exo-go/src/util"
+	"github.com/fatih/color"
+)
+
+// Logger represents a logger
+type Logger struct {
+	Roles         []string
+	SilencedRoles []string
+	Length        int
+	Colors        map[string]func(string, ...interface{})
+}
+
+// NewLogger is Logger's constructor
+func NewLogger(roles, silencedRoles []string) *Logger {
+	logger := &Logger{Roles: roles, SilencedRoles: silencedRoles}
+	logger.Colors = map[string]func(string, ...interface{}){
+		"exocom":     color.Cyan,
```

Comment:
Why do we hard-code colors here? Let's simply assign the first color to the first role, the second color to the second role, etc.